src/app/shared/models/child.model.ts

```diff
@@ -13,8 +13,8 @@ export class Child {
   constructor(info) {
     this.firstName = info.firstName;
     this.lastName = info.lastName;
-    this.middleName = info.second;
-    this.birthDay = "2021-04-27";
+    this.middleName = info.middleName;
+    this.dateOfBirth = "2021-04-27";
```

Comment:
why do we set value in the model?

Reply:
I am not clear with the date format, I think we should discuss one standard format for all dates, and as a result, we will not have problems later with its conversion.

Here it is temporary so children can be created. Otherwise, the data validation fails.